nixos/tests/all-tests.nix

```diff
@@ -105,7 +105,7 @@ in {
 
   _3proxy = runTest ./3proxy.nix;
   aaaaxy = runTest ./aaaaxy.nix;
-  acme = runTest ./acme.nix;
+  acme = import ./acme/default.nix { inherit runTest; };
```

Comment:
I'm not very familiar with how this is structured, but it looks like there's already established patterns for nesting tests, see `installed-tests` and `incus{,-lts}` for instance. Can we follow that?

Reply:
I wouldn't really call them established patterns. Both of them do pretty weird stuff wrt setting up a custom test derivation builder. I just wanted to place a bunch of independent tests under a subdirectory. I will ask in the contributors matrix chat for a general review of this.